src/serena/tools/config_tools.py

```diff
@@ -2,6 +2,19 @@
 from serena.tools import Tool, ToolMarkerDoesNotRequireActiveProject, ToolMarkerOptional
 
 
+class OpenDashboardTool(Tool, ToolMarkerDoesNotRequireActiveProject):
```

Comment:
should inherit from `ToolMarkerOptional` and documented that the tool should be explicitly enabled through the config. I suspect most users won't want it in their context window